src/main/java/com/launchacademy/giantleap/models/ItemReview.java

```diff
@@ -0,0 +1,56 @@
+package com.launchacademy.giantleap.models;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.validator.constraints.URL;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="fashion_item_reviews")
+@Getter
+@Setter
+@NoArgsConstructor
+public class ItemReview  {
```

Comment:
if you're calling your class FashionItem I would suggest naming this FashionItemReview for clarity's sake.